ironfish-rust/src/transaction/tests.rs

```diff
@@ -40,7 +48,13 @@ fn test_transaction() {
         "A really cool coin",
     )
     .expect("should be able to create an asset");
-    let mint_out_note = Note::new(receiver_key.public_address(), 2, "", asset.generator());
+    let mint_out_note = Note::new(
+        receiver_key.public_address(),
+        2,
+        "",
+        asset.generator(),
+        spender_key.public_address(),
```

Comment:
Should this be spender or sender?

Reply:
For the mint out note, this sender address is the the spender's address